x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/details/host_details.tsx

```diff
@@ -77,14 +77,13 @@ export const HostDetails = memo(({ details }: { details: HostMetadata }) => {
   const [policyResponseUri, policyResponseRoutePath] = useMemo(() => {
     const { selected_host, show, ...currentUrlParams } = queryParams;
     return [
-      getManagementUrl({
+      getEndpointDetailsPath({
```

Comment:
For this here adn below (line 95) - it does not look like we need two separate variables to hold the routePath since the URL pathname and the router Path are the same. We should collapse it. 
We can correct this once you merge this PR in :) 

//cc @kevinlog

Reply:
I think we might need to use the `formatUrl` if the `routerPath` needs to have the full URL.